### ISSUE TITLE
Update Participant Recruitment guidelines 

### ISSUE THREAD

**Issue by naomimarcussen** [department-of-veterans-affairs/va.gov-team#50416]
### Description
Some researchers have had challenges recruiting the right people for their studies (generative, usability and UAT). Perigean has agreed to work more closely with our researchers to ensure they get the right participants for their studies. In particular, according to [this thread](https://dsva.slack.com/archives/C0216PL32HJ/p1670342102708559), the following changes to the recruitment process have been agreed to be updated.

Suggested language:

To ensure you get the right participants for your study, you should:
- Outline any must-have criteria in the screener questions in the research plan, even if they are already outlined in the recruitment criteria table.

- Request a kick-off call with Perigean so that you can: 
- - Review the Calendly survey and the screener email before sent to participants
- - Ask Perigean to follow up with participants before sessions to verify that they are able to meet the technical requirements (i.e. they have Zoom downloaded, can screen share, and can log in to VA.gov for the session)

To request a kick-off call, add the requested date/time in the research plan and allow for extra time so that there are 5 business days after the call for recruiting. Then during the kick-off call, Perigean will screen share so the researcher can review the survey and email content they plan on sending participants.

@naomimarcussen will work with Shane Strassberg to make sure the language is accurate.

### Relevant URLs
Section 3 of the [Research Checklist](https://depo-platform-documentation.scrollhelp.site/research-design/research-checklist#ResearchChecklist-1.Planningforresearch)

**Comment by laucon** [comment]
@naomimarcussen Questions:
1. Do we want to add this information about Participant Recruitment guidelines into a Platform website document?
2. 1. Is this in any way related to [Robyn's ticket here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/50386)?

**Comment by naomimarcussen** [comment]
@laucon Answers:
1. Yes, we can add this language to Section 3 of the [Research Checklist](https://depo-platform-documentation.scrollhelp.site/research-design/research-checklist#ResearchChecklist-1.Planningforresearch) - sorry I didn't explain that very well
2. No, this is not related to [Robyn's ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/50386)

**Comment by laucon** [comment]
hey @naomimarcussen! Since I'm writing this info into the document, I wanted to make sure of a few things before I do that. There are a lot of words that I don't know about in the document, but that the people reading the document will probably know about. Will the people reading the document know...
1. what screener questions are
2. what a research plan is
3. what a recruitment criteria table is
4. what Perigean is
5. what a Calendly survey is
6. what a screener email is
7. what a kick-off call is

I also want to ask 2 more questions:
8. The way I understand it, it sounds like a kick-off call is NOT the same as the actual user interview. Rather, the kick-off call is big step that happens BEFORE the user interview. Is that true? And if so, will readers of this document know that? I just want to make sure that everything is as clear as it can be.
9. You wrote "Review the Calendly survey and the screener email before sent to participants." Are the Calendly survey AND the screen email BOTH going to be sent to participants or is ONLY the screener email going to be sent to participants?

**Comment by naomimarcussen** [comment]
@laucon Thank you for verifying this. I've taken another stab at this language and how it fits into the Research checklist page and will have Shane review it.

**Comment by laucon** [comment]
@naomimarcussen Fantastic. by the way, I think the writing is great. I just know I'm a total outsider to this process and I'm not sure how much inner knowledge the people reading the articles will have, so I want to make sure that it's clear for the right audience. :)

**Comment by naomimarcussen** [comment]
@laucon I've reviewed the copy above with Shane and made edits based on his feedback. It's good to add to a draft of the Research checklist. Ping me when you have that in and I'll have Shane review it once more before we publish.

**Comment by laucon** [comment]
@naomimarcussen I agree that this is a lot clearer. You wrote, "This is an option offered by Perigean so that you can **review the screener questions and email that will be sent** to participants before it's sent out." I had to read this sentence 3 times. Are you saying that the UX researcher can review 2 things: they can review the screener questions and they can review the email that is about to be sent out? If so, can we make this a clearer sentence by writing it as "This is an option that Perigean offers so that you can read over both the screener questions and the email before that email is sent out."

**Comment by naomimarcussen** [comment]
Yes! That's much better than mine. Thank you

**Comment by laucon** [comment]
Happy to help!

**Comment by laucon** [comment]
@naomimarcussen I added all the changes. It will be published around 4:30pm today. So I suggest you get Shane to check I quick. lol. Also, I added each line to the bottom of the respective sections. For example, "Write screener questions" and "Request a kick-off call with Perigean" are now below all the other paragraphs within Step 1. Ditto with Step 3. So if you'd like to change the order, that's fine. Just let me know.

**Comment by naomimarcussen** [comment]
@laucon Thank you for getting on that so quick! Thanks for the heads up on the order as well. I tweaked the order of the check box items to make sense logically. 

I see it got moved back to the Oops folder for totally other reasons, which I've fixed. For the next go around, since I have to go back-and-forth with Shane, it's easier if we keep it in the drafts folder until I have the final go ahead, then move it to be published. I've sent him the draft for final review and am waiting for his response.

**Comment by laucon** [comment]
@naomimarcussen perfect. I think that makes perfect sense. If you need me, I'm here. Otherwise, feel free to go ahead and do your thing. lol

**Comment by laucon** [comment]
@naomimarcussen Oh, but just so you know, if it is in the drafts folder, people aren't going to be able to see the document on the Platform website until it's back out of the drafts folder. If that's a problem, we can make a temporary copy of it to keep published with all the major launch blocking issues fixed and then you can publish the more finished version youre ready. But if youre ok with there being a blank page on the Platform website for a few days while you edit, thats fine with us.

**Comment by naomimarcussen** [comment]
Oh! Thanks for the heads up. I guess I assumed the version in the drafts folder was a copy. I'll probably hear back from Shane pretty quickly and it should be back in Platform website tomorrow.

**Comment by laucon** [comment]
@naomimarcussen well, we're not gonna publish until Monday at 3:30pm. so either way, you'll probably have to edit it on Friday or on Monday up until 3:30 and then we can republish it. But I'm in charge of deploy on Mondays, so just let me know and I can take it out of the drafts folder by then and publish it. and also another note: I'm gonna be leaving my apartment at 5:30pm Monday night, so if there are any issues after that, I won't be available until the next morning (they've got some of the biggest stars on Broadway singing a concert version of Chess the musical Monday night. woo hoo! haha)

**Comment by naomimarcussen** [comment]
Hi! I've reviewed the draft page with Shane and we're all good to go! I've moved it back to the Platform website folder.

Thanks for all your help!
